_layouts/default.html

```diff
@@ -27,12 +27,22 @@
         {% endif %}
 
         {% if site.show_downloads %}
-          <ul>
+          <ul id="download">
             <li><a href="{{ site.github.zip_url }}">Download <strong>ZIP File</strong></a></li>
             <li><a href="{{ site.github.tar_url }}">Download <strong>TAR Ball</strong></a></li>
             <li><a href="{{ site.github.repository_url }}">View On <strong>GitHub</strong></a></li>
           </ul>
         {% endif %}
+        <ul>
+          <li>This is an unordered list in the header.</li>
+          <li>This is an unordered list in the header.</li>
+          <li>This is an unordered list in the header.</li>
+        </ul>
+        <ol>
+          <li>This is an ordered list in the header.</li>
+          <li>This is an ordered list in the header.</li>
+          <li>This is an ordered list in the header.</li>
+        </ol>
```

Comment:
I don't believe this placeholder text should be part of the theme. The theme should work without any configuration on the part of the user. If a user wants links in the sidebar, we should either make it a configuration setting, and document how to set it, or require them to modify the template directly.